articles/channel.html

```diff
@@ -301,7 +301,7 @@ <h3>Channel Operations</h3>
 
 	<p>
 	</p>
-	Two channel receive operations which are used as source values in assignments:
+	Two channels receive operations which are used as source values in assignments:

```

Comment:
Ah, I think here the "Two" is a qualifier of "operations". ;D

Reply:
Ah, that makes sense. Reverted the change.